extension/registry/holixon/holixon-axon/src/main/kotlin/api/RegisterAvroSchemaCommand.kt

```diff
@@ -0,0 +1,15 @@
+package io.holixon.avro.adapter.registry.axon.api
+
+import org.axonframework.modelling.command.TargetAggregateIdentifier
+
+/**
+ * Registers the schema.
+ */
+data class RegisterAvroSchemaCommand(
+  @TargetAggregateIdentifier
+  val schemaId: String,
```

Comment:
Design decision: each schema evolution has a new id. Wouldn't a namespace+name identifier (or a random uuid) would be a better fit to track evolution of a single schema? with the fingerprint id, there is no activity on an aggregate besides "register".

Taking this idea one step further, the aggregate could check the compatibility of a new schema revision for the same identifier, reject registration based on incompatibilities or add the compatibility state to the event, so the projection knows about multiple revisions and state.